pkg/client-sdk/example/main.go

```diff
@@ -0,0 +1,157 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os/exec"
+	"strings"
+	"sync"
+
+	arksdk "github.com/ark-network/ark-sdk"
+	log "github.com/sirupsen/logrus"
+)
+
+func main() {
+	var (
+		explorerUrl = "http://localhost:3001"
+		network     = "regtest"
+		aspUrl      = "http://localhost:6000"
+		aspPubKey   = "ASP_PUBKEY"
+
+		ctx         = context.Background()
+		explorerSvc = arksdk.NewExplorer(explorerUrl)
+	)
+
+	alice, err := arksdk.NewSingleKeyWallet(
+		explorerSvc, network, "",
+	)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	aliceArkClient, err := arksdk.New(
+		alice, explorerUrl, arksdk.Grpc, network, aspUrl, aspPubKey,
+	)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	if err := aliceArkClient.Connect(ctx); err != nil {
+		log.Fatal(err)
+	}
+
+	_, aliceOnchainAddr, err := aliceArkClient.Receive(ctx)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	_, err = runCommand("nigiri", "faucet", "--liquid", aliceOnchainAddr)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	txID, err := aliceArkClient.Onboard(ctx, 20000)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	log.Infof("Alice onboarded with txID: %s", txID)
+
+	bob, err := arksdk.NewSingleKeyWallet(
+		explorerSvc, network, "",
+	)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	bobArkClient, err := arksdk.New(
+		bob, explorerUrl, arksdk.Grpc, network, aspUrl, aspPubKey,
+	)
+
+	if err := bobArkClient.Connect(ctx); err != nil {
+		log.Fatal(err)
+	}
+
+	bobOffchainAddr, _, err := aliceArkClient.Receive(ctx)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	txID, err = aliceArkClient.SendOffChain(
+		ctx,
+		false,
+		[]arksdk.Receiver{
+			{
+				To:     bobOffchainAddr,
+				Amount: 10000,
+			},
+		})
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	log.Infof("Alice sent 10000 to Bob offchain with txID: %s", txID)
+}
```

Comment:
same here, let's mine a block and check the balances of both wallets

Reply:
Done

Reply:
Done